packages/css/src/components/menu/menu.scss

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright Gemeente Amsterdam
+ */
+
+@use "../../common/hyphenation" as *;
+@use "../../common/text-rendering" as *;
+@use "../../common/breakpoint" as *;
+
+@mixin reset-ul {
+  list-style: none;
+  margin-block: 0;
+  padding-inline-start: 0;
+}
+
+.ams-menu__list {
+  box-sizing: border-box;
+  display: grid;
+
+  @include hyphenation;
+  @include text-rendering;
+  @include reset-ul;
+}
+
+.ams-menu__link {
+  align-items: flex-start;
+  color: var(--ams-menu-link-color);
+  display: flex;
+  font-family: var(--ams-menu-link-font-family);
+  font-size: var(--ams-menu-link-font-size);
+  font-weight: var(--ams-menu-link-font-weight);
+  gap: var(--ams-menu-link-gap);
+  line-height: var(--ams-menu-link-line-height);
+  outline-offset: var(--ams-menu-link-outline-offset);
+  padding-block: var(--ams-menu-link-padding-block);
+  text-decoration-line: var(--ams-menu-link-text-decoration-line);
+  text-decoration-thickness: var(--ams-menu-link-text-decoration-thickness);
+  text-underline-offset: var(--ams-menu-link-text-underline-offset);
+
+  &:hover {
+    color: var(--ams-menu-link-hover-color);
+    text-decoration-line: var(--ams-menu-link-hover-text-decoration-line);
+  }
+
+  // TODO: Discuss className specificity in child components
+  .ams-menu__icon {
```

Comment:
This needed to be nested like this because the specificity of was the same as `.ams-icon` and got overruled.

Reply:
Right but i'm not working in the icon.scss file right?

Reply:
To elaborate I do not want to target `.ams-icon` in this file. To target the className modifier on the `Icon` component inside `.ams-menu__link` seems like the way to go.